app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(cp: str) -> None:
+    parts = cp.split(" ")
+    old = parts[1]
+    new = parts[2]
+    if old == new:
+        return
+    else:
+        with open(old, "r") as file_in:
+            with open(new, "w") as file_out:
```

Comment:
You can simplify the file opening process by using a single `with` statement with a comma to open both files simultaneously, as suggested in the task description.